package.json

```diff
@@ -7,7 +7,7 @@
     "test": "node tests/test_template_generator.js"
   },
   "dependencies": {
-    "assemblyscript": "lrettig/assemblyscript#ewasm"
+    "assemblyscript": "github:holochain/assemblyscript"
```

Comment:
Created this fork in the github org. We just have to remember to pull updates from the upstream occasionally. We should aim to move this back to the original once it is stable